render_dashboard.js

```diff
@@ -11,10 +11,8 @@
 // Some constants.
 //
 
-var DOCUMENT_WIDTH = 720;
-var DOCUMENT_HEIGHT = 3000;
-
-
+var DOCUMENT_WIDTH = (screen.width*200)/100;
+var DOCUMENT_HEIGHT = (screen.height*200)/100;
```

Comment:
Hey, you are right there's no real logic behind, I just put that because it works fine for me.
 
We can make it configurable yes or put a hardcoded value like before.